sources/FSUtils.cpp

```diff
@@ -77,12 +81,13 @@ status_t CheckCopiable(BEntry* src, BEntry* dest)
 		// We have an existing file, so query the user what to do.
 		status_t returncode;
 
-		newpath = "The file ";
-		newpath += name;
-		newpath += " exists. Do you want to replace it?";
+		newpath = B_TRANSLATE_COMMENT("%name% already exists. Do you want to "
```

Comment:
I suggest to place the %name% variable in single quotes: '%name%' already exists...
If the filename is "Invoice_0887-17.docx" it's obvious. If the filename is "Something", things are less so... :)